lib/bolt_spec/plans/action_stubs/script_stub.rb

```diff
@@ -31,7 +31,9 @@ def call(targets, script, arguments, options)
       end
 
       def parameters
-        @invocation[:arguments] + @invocation[:options] if @invocation.include[:arguments]
+        # For scripts, arguments are an array while options are a hash
+        # Merging the two requires adding an arguments key to hold the array of arguments for error printing
+        @invocation[:options].merge('arguments' => @invocation[:arguments]) if @invocation.include?(:arguments)
```

Comment:
Given these seem to only be generated for helpful error messages, maybe we should standardize by either returning a hash or an interpolated string for all the different action stubs.

Reply:
The three other action stubs are already returning hashes, so this change aligns the script stub with the others.

It looks like the `parameters` method is just rebuilding the original hash passed to `with_params`. Would it make more sense to just store this original hash and return it instead of rebuilding it? That's essentially what the command and upload stubs do.

Reply:
Yeah, If we go the hash route i would suggest having a hash that always has the same two keys. Something like `{options: @invocation[:options], arguments: @invocation[:arguments]}` then the caller would know what to do with that hash.